libs/platform/src/lib/components/feed-input/feed-input.component.spec.ts

```diff
@@ -44,7 +44,7 @@ describe('FeedInputComponent', () => {
         component.disabled = true;
         fixture.detectChanges();
 
-        expect(textareaEl.nativeElement.getAttribute('aria-disabled')).toEqual('true');
+        expect(textareaEl.nativeElement.getAttribute('ng-reflect-is-disabled')).toEqual('true');
```

Comment:
Can't you check for `disabled` attribute? 
```
expect(textareaEl.nativeElement.disabled).toBeTruthy();
```

Reply:
fixed